### PR TITLE
Fix link to overview to avoid 404 in Drupal

### PR DESCRIPTION
**Pull request by enzolutions**
Fix for https://github.com/govCMS/govCMS8/issues/148

### CODE REVIEW

**Review by fubarhouse — APPROVED** [review]
The changes are as described and tie in with changes from govCMS/govCMS8/pull/161.

Note that this change has been recompiled and included to remove needless space characters from empty lines (linting), which has made this PR much larger than it should otherwise have become.

File: styleguide/index.html [hunk 1]
```diff
@@ -43,10 +43,10 @@ <h1 class="kss-doc-title">GovCMS 8 UI-Kit Starter Style Guide</h1>
         <div id="kssref-0" class="kss-section kss-section--depth-0 kss-overview kss-style">
       <h1>GovCMS 8 UI-Kit Starter Styleguide</h1>
 
-<p>This styleguide is showing only the UI-Kit components with a reasonable 
-template. It is not providing a comprehensive list of every style 
+<p>This styleguide is showing only the UI-Kit components with a reasonable
+template. It is not providing a comprehensive list of every style
 available in UI-Kit or the theme. Feel free to add other components as needed.</p>
-<p>For live examples of other styling, install the defaul content and visit 
+<p>For live examples of other styling, install the defaul content and visit
```
Comment (gargsuchi) [inline styleguide/index.html]: @enzolutions Can you please fix the typo in this line while you are at it - defaul -> default

cc: @fubarhouse

Comment (enzolutions) [reply styleguide/index.html]: done

**Review by fubarhouse — APPROVED** [review]

**Comment by enzolutions** [comment]
<3

**Pull request merged**